test/direct-message-key.test.js

```diff
@@ -1,7 +1,15 @@
 /* eslint-disable camelcase */
 
 const test = require('tape')
-const { generate } = require('ssb-keys')
+const ssbKeys = require('ssb-keys')
+const bbKeys = {
+  generate () {
+    const keys = ssbKeys.generate()
+    keys.id = keys.id.replace('ed25519', 'bbfeed-v1')
+    return keys
+  }
+}
+
```

Comment:
added this function.
would be nice if this existed somewhere else that we could import